docs/basso-livello/medical.md

```diff
@@ -0,0 +1,206 @@
+---
+template: home.html
+title: Sistema Medico
+---
+
+# Sistema Medico
+
+Questa sezione copre la standardizzazione del sistema medico usato dal Network Bagigi.
+
+Le informazioni in questa guida fanno riferimento alle documentazioni delle mod [ACE Medical](https://ace3.acemod.org/wiki/feature/medical-system) e [KAT Advanced Medical](https://docs.google.com/document/d/1YlY00qoL2gdrbOJJzi5aW19A3HqT7uXflszDYuK7h3M/edit).
+
+## Concetti generali
+
+A prescindere dalle mod e i settaggi esatti in uso, esistono dei concetti generali utili a medicare altri giocatori in missione in maniera efficiente.
+
+### Il personale medico
+
+#### Il Fante
+
+Un fante senza specializzazione può a volte stabilizzare un ferito senza richiedere l'intervento di un soccorritore/medico, ma anche nei casi in cui non gli è possibile sarà comunque in grado di alleggerire il compito al personale medico.
```

Comment:
Un fante senza specializzazione può a volte stabilizzare un ferito senza richiedere l'intervento di un soccorritore o di un medico. Nei casi in cui questo non sia possibile, può comunque alleggerire il lavoro del personale medico.